torch/_C/_distributed_c10d.pyi

```diff
@@ -544,19 +544,23 @@ class _ProcessGroupWrapper(Backend):
     wrapped_pg: Backend
 
 class ProcessGroupNCCL(Backend):
-    class Options:
-        def __init__(self, timeout: timedelta | None = None) -> None: ...
-        @property
-        def backend(self) -> str: ...
-        @property
-        def _timeout(self) -> timedelta: ...
-        @_timeout.setter
-        def _timeout(self, val: timedelta) -> None: ...
+    class NCCLConfig:
+        blocking: int
+        cga_cluster_size: int
+        min_ctas: int
+        max_ctas: int
+
+        def __init__(self) -> None: ...
         @property
-        def _is_high_priority_stream(self) -> bool: ...
-        @_is_high_priority_stream.setter
-        def _is_high_priority_stream(self, val: bool) -> None: ...
+        def net_name(self) -> str: ...
 
+    class Options:
```

Comment:
did we delete the `_timeout` intentionally from NCCLOptions?

Reply:
I checked the bindings and did not see timeout